src/autocomplete/src/AutocompleteItem.js

```diff
@@ -1,6 +1,98 @@
 import React, { memo, forwardRef } from 'react'
 import PropTypes from 'prop-types'
-import Option from '../../select-menu/src/Option'
+import { useListBehavior, useStyleConfig } from '../../hooks'
+import { Image } from '../../image'
+import { Pane } from '../../layers'
+import TextTableCell from '../../table/src/TextTableCell'
+
+const disableProps = { color: 'muted' }
+const selectedProps = { color: 'selected' }
+const emptyProps = {}
+
+const pseudoSelectors = {
+  _focus: ':focus',
+  _hover: ':hover',
+  _active: '&[aria-current="true"]:active, &[data-isselectable="true"]:active',
+  _current: '&[aria-current="true"]',
+  _isSelectable: '&[data-isselectable="true"]'
+}
```

Comment:
Would `_disabled` be useful here?

Also, `_isSelectable` actually seems like the less standard pseudo – maybe we can remove it if we respect enabled vs disabled options?